### PR TITLE
Update the documentation for ReasonEntryViolatesConstraints.

### PR DESCRIPTION
**Pull request by angelazhang8**
Corresponding documentation change for cl/579986405 that refactors ReasonEntryViolatesCOnstraints to support action constraints.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,14 +82,15 @@ absl::variant<ConstraintInfo, std::vector<absl::Status>> P4ToConstraintInfo(
...
-// associated table. Returns the empty string if this is the case, or a
-// human-readable nonempty string explaining why it is not the case otherwise.
-// Returns an InvalidArgument Status if the entry belongs to a table
-// not present in ConstraintInfo, or if it is inconsistent with the table
-// definition in ConstraintInfo.
+// Checks if a given table entry satisfies the entry/action(s) constraint(s)
+// attached to its associated table/action(s). Returns the empty string if this
+// is the case or a human-readable nonempty string explaining why it is not the
+// case otherwise. If the entry meets the table constraint and no action is
+// present, an empty string is returned. Returns an InvalidArgument Status if
```
Comment (smolkaj) [inline README.md:89]: "If the entry meets the table constraint and no action is present, an empty string is returned."

We already said this one sentence earlier, remove?

**Review by smolkaj — APPROVED** [review]
Thank you for updating the documentation!

File: README.md (hunk 1 above)
Comment (angelazhang8) [reply README.md]: I see how you can infer this from the first sentence. Removed.

**Review by smolkaj — APPROVED** [review]
Beautiful!

**Pull request merged**